iot-hub/Samples/device/DeviceReconnectionSample/DeviceReconnectionSample.cs

```diff
@@ -49,8 +50,6 @@ public DeviceReconnectionSample(List<string> deviceConnectionStrings, TransportT
 
             _transportType = transportType;
             _logger.LogInformation($"Using {_transportType} transport.");
-
-            InitializeClient();
```

Comment:
calling this within `RunSampleAsync`, it doesn't need to be in the constructor necessarily.